quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/Environment.java

```diff
@@ -41,6 +39,8 @@
     public static final String ENV_PROFILE ="KC_PROFILE";
     public static final String DATA_PATH = "/data";
     public static final String DEFAULT_THEMES_PATH = "/themes";
+    public static final String DEV_PROFILE_VALUE = "dev";
+    public static final String USER_INVOKED_CLI_COMMAND = "picocli.invoked.command";
```

Comment:
Don't we have already CLI_ARGS ? Any reason why we need another one?

I would also avoid having our sys props using a prefix other than `kc.`.